project/study/models.py

```diff
@@ -163,6 +164,14 @@ def copy_across_assessment(cls, studies, assessment):
             for child in children:
                 child.copy_across_assessments(cw)
 
+        # epi-meta single_results contain cross-study data, so we need to finish
+        # copying studies before copying single results.
+        if studies[0].epi_meta:
```

Comment:
this only checks if the first study in the list is ``epi_meta`` right? what if the first study isn't epi meta but the second is?